language/bytecode-verifier/bytecode-verifier-tests/src/unit_tests/generic_ops_tests.rs

```diff
@@ -0,0 +1,738 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
Nice tests!

FWIW these can be expressed in the IR. But if you prefer these, dealers choice

Reply:
interesting... I should have known given the test error that was there....
I'll probably leave it as is for now...